orbit/pkg/packaging/windows.go

```diff
@@ -149,19 +149,59 @@ func BuildMSI(opt Options) (string, error) {
 		}
 	}
 
-	if err := wix.Heat(tmpDir, opt.NativeTooling, opt.LocalWixDir); err != nil {
+	absWixDir := opt.LocalWixDir
+	wineChecked := false
+
+	// Download wix for macOS running on arm64, unless a local-wix-dir is provided
+	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" && absWixDir == "" {
+		const wixDownload = "https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip"
```

Comment:
Could we also download WiX for generating MSIs from Windows devices?

Reply:
Yes, we can. @noahtalerman, should I file a story for using native build on Windows by default?

One problem is that Windows already allows this with a paid feature, `--local-wix-dir`, so making it free would conflict with that.

Also, we can view this PR as a temporary fix for macOS arm64. We should be able to return to Docker when we can run an arm64 docker image running an arm64 wine. In theory, this is possible now, but I haven't gotten it to work, and haven't gotten a real root cause for the issues. The next step would be trying to get wine to use https://fex-emu.com/